tests/Integration/Behaviour/Features/Scenario/Tax/tax_management.feature

```diff
@@ -0,0 +1,56 @@
+@reset-database-before-feature
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s tax
+Feature: Manage tax
+  As a employee
+  I must be able to correctly add, edit and delete tax
+
+  Scenario: Adding new tax
+    When I add new tax "tax-1" with following properties:
+      | name         | my custom tax 500 |
+      | rate         | 0.5               |
+      | is_enabled   | 1                 |
+    Then tax "tax-1" name in default language should be "my custom tax 500"
+    And tax "tax-1" rate should be 0.500
+    And tax "tax-1" should be enabled
+
+  Scenario: Editing tax
+    When I edit tax "tax-1" with following properties:
+      | name         | my custom tax 300 |
+      | rate         | 0.15              |
+      | is_enabled   | 0                 |
+    Then tax "tax-1" name in default language should be "my custom tax 300"
+    And tax "tax-1" rate should be 0.150
+    And tax "tax-1" should be disabled
+
+  Scenario: Editing tax when providing only changeable property should be allowed
+    When I edit tax "tax-1" with following properties:
+      | name        | tax for fun       |
+    Then tax "tax-1" name in default language should be "tax for fun"
+    And tax "tax-1" rate should be 0.150
+    And tax "tax-1" should be disabled
+
+  Scenario: Toggling tax status
+    Given tax "tax-1" should be disabled
+    When I toggle tax "tax-1" status
+    Then tax "tax-1" should be enabled
+
+  Scenario: Disabling taxes status in bulk action
+    Given taxes with ids: "1,2,3" exists
```

Comment:
Mmmm 😐 it's sad we have a lot of good scenarios talking about tax references (like `tax-1` although it could be interesting to use names even closer to reality like "TVA", "State Tax", "IRS tax") and suddenly we have scenarios talking about IDs (which are technical-oriented)

Cant we create 3 taxes before using the bulk ?

```
When I add new tax "tax-1" with following properties:
      | name         | my custom tax 501 |
      | rate         | 0.5               |
      | is_enabled   | 1                 |
And When I add new tax "tax-2" with following properties:
      | name         | my custom tax 502 |
      | rate         | 0.6               |
      | is_enabled   | 1                 |
And I add new tax "tax-3" with following properties:
      | name         | my custom tax 503 |
      | rate         | 0.7               |
      | is_enabled   | 1                 |
And I disable the following taxes: "tax-1", "tax-2", "tax-3"
```

And we retrieve the IDs required for the Command from the tax references